hilti/runtime/src/fiber.cc

```diff
@@ -249,11 +249,11 @@ std::pair<char*, char*> detail::StackBuffer::activeRegion() const {
     // The direction in which the stack grows is platform-specific. It's
     // probably gong to be growing downwards pretty much everywhere, but to be
     // safe we whitelist platforms that we have confirmed to do so.
-#if __x86_64__
+#if __x86_64__ || __arm64__
```

Comment:
Looking at e.g., https://stackoverflow.com/a/664779/176922, it seems that ARM has no fixed stack growth direction. I think we should either detect the growth direction during the configure phase, or use a more restricted selection of platforms (e.g., if you know Apple M1 grows down, explicitly select for that here instead of `__arm64__`).

Reply:
Yeah, pretty sure it's that way here, but you're right on ARM in general. Need to figure out how to test for M1 specifically then.

Reply:
Done